build.js

```diff
@@ -238,33 +237,37 @@ function githubLinks (options) {
 }
 
 // This function builds the layouts folder for all the Stylus files.
-function buildLayouts () {
-  console.log('[metalsmith] build/layouts started')
-  const labelForBuild = '[metalsmith] build/layouts finished'
+function buildCSS () {
+  console.log('[stylus] static/css started')
+  const labelForBuild = '[stylus] static/css finished'
   console.time(labelForBuild)
 
   fs.mkdir(path.join(__dirname, 'build'), () => {
-    fs.mkdir(path.join(__dirname, 'build', 'layouts'), () => {
-      const metalsmith = Metalsmith(__dirname)
-      metalsmith
-        // Sets the build source as /layouts/css.
-        .source(path.join(__dirname, 'layouts', 'css'))
-        // Deletes Stylus partials since they'll be included in the main CSS
-        // file anyways.
-        .use(filterStylusPartials())
-        .use(stylus({
-          compress: process.env.NODE_ENV !== 'development',
-          paths: [path.join(__dirname, 'layouts', 'css')],
-          use: [autoprefixer()]
-        }))
-        // Pipes the generated files into /build/layouts/css.
-        .destination(path.join(__dirname, 'build', 'layouts', 'css'))
+    fs.mkdir(path.join(__dirname, 'build/static'), () => {
+      fs.mkdir(path.join(__dirname, 'build/static/css'), () => {
+        fs.readFile(path.join(__dirname, 'layouts/css/styles.styl'), 'utf8', (err, data) => {
+          if (err) {
+            throw err
+          }
+
+          stylus(data)
+            .set('compress', process.env.NODE_ENV !== 'development')
+            .set('paths', [path.join(__dirname, 'layouts/css')])
+            .use(autoprefixer())
+            .render((error, css) => {
+              if (error) {
+                throw error
+              }
 
-      // This actually executes the build and stops the internal timer after
-      // completion.
-      metalsmith.build((err) => {
```

Comment:
This is probably a stupid question since I haven't worked with this project & metalsmith in a good while; is it deliberate to stop invoking `metalsmith.build()`?

This line removes it, but doesn't re-introduce it, right?

Reply:
That's one of the things this patch solves. We don't need metalsmith in order to build the CSS, we can just use stylus directly.

Not sure how things were in the past, but as it is now, this was redundant.

Reply:
Huh, alrighty! I gotta take a deeper dive to see how that actually works then I guess.